imageprocessor/views.py

```diff
@@ -139,3 +144,31 @@ def register(request):
         return HttpResponseRedirect(reverse('login'))
     context['form'] = form
     return render(request, 'register.html', context)
+
+class ClassifyAPI(APIView):
+
+    def post(self, request, format=None):
+        response_data = {
+            'url' : None,
+            'tags' : []
+        }
+        form = ImageForm(request.POST or None, request.FILES or None)       
+        try:
+            image_file = request.FILES['file']
+            image = Image.open(image_file)
+            tags = detect(image) 
+            if (tags):
+                try:
+                    response_data['tags'] = tags
+                    current_res = upload_image_to_cloudinary(image_file, tags)
+                    response_data['url'] = current_res.get('url', '') or None
```

Comment:
You're using two fallbacks here. In the object.get() function, the second argument is a default value in case the call fails. It'd be simpler, more concise, and produce the same result if you used None as the default value
i.e. `response_data['url'] = current_res.get('url', None)`

Reply:
Thanks for the tip. I'll make this change